### ISSUE TITLE
Aurora MySQL Data API - migration fails on knex_migrations insert step

### ISSUE THREAD

**Issue by vdimitrieski** [markusahlstrand/knex-data-api-client#62]
# Environment

Knex version:  "0.95.6",
Database + version: Aurora MySQL 5.7 Serverless accessed through Data API
Client: "knex-aurora-data-api-client": "1.5.4",

# Bug

When trying to run a migration to create a simple table, it fails on inserting a tuple into the knex_migrations table. After the migration failure, the required table is created and there is a row in the knex_migrations_lock table, however the migration fails with the error given below. If created manually, data can be read from the table with the same knex connection parameters and therefore the connection is set up well. The error:
```
migration file "20210602103458_create_example_table.js" failed
migration failed with error: insert into `knex_migrations` (`batch`, `migration_time`, `name`) values (1, '2021-06-02 13:06:32.145', '20210602103458_create_example_table.js') - Database error code: 1292. Message: Data truncation: Incorrect datetime value: '"2021-06-02T11:06:32.145Z"' for column 'migration_time' at row 1
```

You can basically try to create any table. I created a simple table with 4 int columns in an empty database and still got the error. However, if you manually run the insert from the error on the database, it works.

**Comment by markusahlstrand** [comment]
Hi @vdimitrieski ,

Thanks for reporting the issue. The test for migrations and mysql is currently commented out and I can't remember when and why this happened.. I'll see if I can get it passing again.

**Comment by markusahlstrand** [comment]
I get this error for the migrations test:
```
data-api-mysql
       knex-migrate
         should setup a database with knex-migrate:
     insert into `knex_migrations` (`batch`, `migration_time`, `name`) values (1, '2021-06-02 16:02:46.063', '20191220234943_init.js') - Database error code: 1292. Message: Data truncation: Incorrect datetime value: '"2021-06-02T14:02:46.063Z"' for column 'migration_time' at row 1
```

So, I guess it has something to do with differences in timestamps formats in mysql vs postgres?

**Comment by vdimitrieski** [comment]
@markusahlstrand thanks for checking out. If you take a look at the error, I think the knex-generated insert command contains a well formatted datetime '2021-06-02 16:02:46.063', but when it reaches Aurora MySQL it is actually in the format '2021-06-02T14:02:46.063Z' which bounced. As this knex-generated insert command is valid, and you can actually can execute it in the Aurora query builder, I assume the error is somewhere in the middle :) 

Any thoughts of what it might be?

**Comment by markusahlstrand** [comment]
Seems like it's being passed as a datetime and we're not handing it correctly when converting it to a sql string. Should hopefully be a quick fix

**Comment by markusahlstrand** [comment]
Published 1.5.5 with a fix that at least makes the migration test happy :) Let me know if it works for you.

**Comment by vdimitrieski** [comment]
It works now with Aurora MySQL as well. Thanks for the fast fix!
